content/millennium/r4/financial/financial-transaction.md

```diff
@@ -35,11 +35,11 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  ID                                      | Value\[x] Type                    | Description
 -----------------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------
  `financial-transaction-account-number`  | [`string`]                        | A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
- `financial-transaction-alias`           | [`string`]                        | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+ `financial-transaction-alias`           | [`string`]                        | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction. The alias needs to be a valid alias in the client domain for the transaction type and that the alias determines the credit or debit impact on the accounts receivable.
```

Comment:
Why is this updated?

Reply:
removing this. Initially change made here but this was not expected change.
https://github.com/cerner/fhir.cerner.com/pull/455/commits/0017aa837e886e8b5c88cc8bf114295d75a54f2e#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR38

Reply:
+1